vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -73,6 +103,8 @@ private void setComponentStyles() {
     projectObjective.setWidthFull();
     experimentalDesignField.setWidthFull();
     formLayout.setClassName("create-project-form");
+    projectManagerComboBox.getStyle().set("--vaadin-combo-box-overlay-width", "16em");
+    projectManagerComboBox.getStyle().set("--vaadin-combo-box-width", "16em");
```

Comment:
NIT: I think down the line it makes sense to move this to a an external css especially once additional comboboxes are added to the page. For now this is fine